client/model/application/info.go

```diff
@@ -0,0 +1,2 @@
+// Package application represents models for external applications/services that CDO interacts or interested in.
+package application
```

Comment:
remove?

Reply:
I want to add some comment for this package for people to understand what this package is for, to do so I need to add this file